File: src/targets/php/http2/fixtures/full.php
<?php

$client = new http\Client;
$request = new http\Client\Request;

$body = new http\Message\Body;
$body->append(new http\QueryString([
  'foo' => 'bar'
]));

$request->setRequestUrl('http://mockbin.com/har');
$request->setRequestMethod('POST');
$request->setBody($body);

$request->setQuery(new http\QueryString([
  'foo' => [
    'bar',
    'baz'
  ],
  'baz' => 'abc',
  'key' => 'value'
]));

$request->setHeaders([
  'accept' => 'application/json',
  'content-type' => 'application/x-www-form-urlencoded'
]);


$client->setCookies([
  'bar' => 'baz',
  'foo' => 'bar'
]);

$client->enqueue($request)->send();
$response = $client->getResponse();

echo $response->getBody();

File: src/targets/r/httr/fixtures/full.r
library(httr)

url <- "http://mockbin.com/har"

queryString <- list(
  foo = "bar,baz",
  baz = "abc"
)

payload <- "foo=bar"

encode <- "form"

response <- VERB("POST", url, body = payload, query = queryString, content_type("application/x-www-form-urlencoded"), accept("application/json"), set_cookies(`foo` = "bar", `bar` = "baz"), encode = encode)

content(response, "text")